bangazonapi/views/profile.py

```diff
@@ -367,5 +391,12 @@ class FavoriteSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Favorite
+        validators = [
```

Comment:
This looks good but instead of using the existing serializer, it would be better to create a new one that's just a `serializers.ModelSerializer` with the `customer` and `seller` fields and no depth.